File: src/tests/Behavioral/BlankImportSideEffects/jpeglike/jpeglike.go
// Package jpeglike is a SECOND blank-imported registrant, so the guard covers more than one blank
// import in a single file: each needs its own generated hook, and two hooks in one class must not
// collide (a shared generated name would be CS0111) nor let the second silently go unforced.
package jpeglike

import "BlankImportSideEffects/registry"

func init() {
	registry.Register("jpeglike", "jpeglike decoder")
}

File: src/tests/Behavioral/BlankImportSideEffects/jpeglike/jpeglike.cs
namespace go.BlankImportSideEffects;

using registry = go.BlankImportSideEffects.registry_package;
using go.BlankImportSideEffects;

partial class jpeglike_package {

[GoInit] internal static void init() {
    registry.Register("jpeglike"u8, "jpeglike decoder"u8);
}

} // end jpeglike_package
